bin/docker-image-tool.sh

```diff
@@ -78,20 +91,38 @@ function build {
   docker build $NOCACHEARG "${BUILD_ARGS[@]}" \
     -t $(image_ref spark) \
     -f "$BASEDOCKERFILE" .
+  if [ $? -ne 0 ]; then
+    error "Failed to build base Docker image, please refer to Docker build output for details"
```

Comment:
Add `.` in the end of the error message to be consistent with the rest of the script. Ditto below.

Reply:
Fixed this and other instances thereof